cms/navigation/tests/factories.py

```diff
@@ -9,11 +9,11 @@
 
 
 class ThemePageChooserFactory(PageChooserBlockFactory):
-    page = factory.SubFactory(ThemePageFactory)
+    page = factory.LazyFunction(lambda: ThemePageFactory().id)
 
 
 class TopicPageChooserFactory(PageChooserBlockFactory):
-    page = factory.SubFactory(TopicPageFactory)
```

Comment:
@sanjeevz3009 these changes are unnecessary. can you remove and then we are good to merge

Reply:
caught up in Slack - `TypeError: Object of type ThemePage is not JSON serializable`